vizro-core/src/vizro/__init__.py

```diff
@@ -2,7 +2,7 @@
 import os
 
 import plotly.io as pio
-from dash.development.base_component import Component, ComponentRegistry
```

Comment:
@antonymilne - this didn't seem to be used. Wondering why the linter didn't remove it by itself. Let me know if we need to import this

Reply:
Hmm yes, no need to import this any more. I wonder why it didn't get removed by `ruff`.